### PR TITLE
HPCC-18169 DOCS:More detail on deschedule/reschedule

### PR DESCRIPTION
**Pull request by g-pan**
Fix HPCC-18169 DOCS:More detail on deschedule/reschedule
Added more detail to the ECL Scheduler book, which by inclusion
will also update the ECL Watch doc.

@JamesDeFabia please review
   Tested:  http://10.240.32.243/view/Docs/job/DocDev0IV/

Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x ] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [x] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

### CODE REVIEW

File: docs/ECLScheduler/ECLSched-Mods/ECLSchedComp.xml [hunk 1]
```diff
@@ -183,6 +183,18 @@
...
+        page in ECL Watch. Select the workunit details page, then press the
+        <emphasis role="bold">Reschedule</emphasis> button to change a
+        selected scheduled workunit. Press the <emphasis
+        role="bold">Deschedule</emphasis> button to stop a selected scheduled
+        workunit from running. </para>
+
+        <para>If you are using a WHEN clause and it contains a COUNT number,
+        when rescheduled the workunit will continue the COUNT from the point
+        where it stopped and resumes whatever COUNT remains. Once a workunit
+        completes all of the COUNT there is no reschedule option.</para>
```
Comment (JamesDeFabia) [inline docs/ECLScheduler/ECLSched-Mods/ECLSchedComp.xml]: Once a workunit completes the COUNT, there is no reschedule option.

File: docs/ECLScheduler/ECLSched-Mods/ECLSchedComp.xml [hunk 2]
```diff
@@ -183,6 +183,18 @@
 
         <para>To view the workunit details, click on the workunit ID (WUID)
         link for the workunit.</para>
+
+        <para>You can modify scheduled workunits from the workunit details
+        page in ECL Watch. Select the workunit details page, then press the
+        <emphasis role="bold">Reschedule</emphasis> button to change a
+        selected scheduled workunit. Press the <emphasis
```
Comment (JamesDeFabia) [inline docs/ECLScheduler/ECLSched-Mods/ECLSchedComp.xml]: to change a selected scheduled workunit. 
s/b 
to reschedule a descheduled workunit.

File: docs/ECLScheduler/ECLSched-Mods/ECLSchedComp.xml [hunk 3]
```diff
@@ -183,6 +183,18 @@
...
+        role="bold">Deschedule</emphasis> button to stop a selected scheduled
+        workunit from running. </para>
+
+        <para>If you are using a WHEN clause and it contains a COUNT number,
+        when rescheduled the workunit will continue the COUNT from the point
+        where it stopped and resumes whatever COUNT remains. Once a workunit
```
Comment (JamesDeFabia) [inline docs/ECLScheduler/ECLSched-Mods/ECLSchedComp.xml]: resumes the remaining COUNT.

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
comments inline

**Comment by g-pan** [comment]
Not sure, what happened here, somehow the wrong commit message hijacked this Pull Req. 
I tried to amend when I squashed the commits, but no luck. I manually edited the message here. @JamesDeFabia  please review. Tested here: http://10.240.32.243/view/Docs/job/DocDev0IV/ 
Still Valid, and to show that build succeeds. where smoketest reports failure.

**Review by JamesDeFabia — APPROVED** [review]
Good to merge

**Pull request merged**